package.sh

```diff
@@ -10,8 +10,17 @@ init_ve() {
 }
 
 package() {
-  sudo rm -r dist
-  pyinstaller --clean --hidden-import=requests main.py
+  local arch=$(uname -m)
+  if [ "$arch" == "x86_64" ]; then
+    if [ -d dist/main ]; then
+      sudo rm -r dist/main
```

Comment:
dont we need to remove dist, instead of dist/main? the initial workflow removed the entire dist directory, just want to make sure we're removing right stuff here

Reply:
we cannot remove dist now. this is because when we are creating aarch64 packages, the dist/main should still contain the x86_64 packages